### ISSUE TITLE
Autosuggest accessibility issues

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2175]
The following accessibility issues with the Autosuggest component have been identified using aXe on the implementation on an [EQ prototype](https://eq-v2-prototypes.herokuapp.com/features/long-list-of-codes/autoselect)...
<img width="393" alt="autosuggest-a11y-1" src="https://user-images.githubusercontent.com/43346934/166952710-c087cad3-c21f-4e86-8d7d-8c5e14f2f93b.png">
<img width="393" alt="autosuggest-a11y-2" src="https://user-images.githubusercontent.com/43346934/166952721-b8242e14-c399-458d-8f32-bbba653ec30e.png">

----
Use of the macro:
```
{{ onsAutosuggest({
        "id": "sector",
        "label": {
        "text": "Sector",
        "description": "Select sector from suggestions",
        "id": "sector-label"
        },
        "autocomplete": "off",
        "instructions": "Use up and down keys to navigate suggestions once you\'ve typed more than two characters. Use the enter key to select a suggestion. Touch device users, explore by touch or with swipe gestures.",
        "ariaYouHaveSelected": "You have selected",
        "ariaMinChars": "Enter 3 or more characters for suggestions.",
        "ariaResultsLabel": "Country suggestions",
        "ariaOneResult": "There is one suggestion available.",
        "ariaNResults": "There are {n} suggestions available.",
        "ariaLimitedResults": "Results have been limited to 10 suggestions. Type more characters to improve your search",
        "moreResults": "Continue entering to improve suggestions",
        "resultsTitle": "Suggestions",
        "resultsTitleId": "sector-suggestions",
        "autosuggestData": "https://api.npoint.io/20ffc805e13c37980073",
        "noResults": "No suggestions found.",
        "typeMore": "Continue entering to get suggestions"
        }) }}
```

**Comment by kruncher** [comment]
I am finding the following when running the `axe` package on this exact autosuggest configuration:
```
 Expected the HTML found at $('#sector-listbox') to have no violations:

    <ul class="ons-autosuggest-input__listbox ons-js-autosuggest-listbox" role="listbox" id="sector-listbox" aria-labelledby="sector-suggestions" tabindex="-1"></ul>

    Received:

    "ARIA input fields must have an accessible name (aria-input-field-name)"

    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
```

**Comment by boxadesign** [comment]
I have performed axe tests on the most recent version of the component and no issues are detected.
